roles/matrix-nginx-proxy/tasks/setup_nginx_proxy.yml

```diff
@@ -21,6 +21,7 @@
     - "{{ matrix_nginx_proxy_base_path }}"
     - "{{ matrix_nginx_proxy_data_path }}"
     - "{{ matrix_nginx_proxy_confd_path }}"
+    - "{{ matrix_nginx_proxy_synapse_cache_path }}"
```

Comment:
This has the potential to attempt to create `/tmp/cache` on the host for certain setups (when `matrix_nginx_proxy_enabled: false`). It should probably be made conditional somehow.

Reply:
I try to make it conditional like so : 
```
- name: Ensure Matrix nginx-proxy paths exist
  file:
    path: "{{ item.directory }}"
    state: directory
    mode: 0750
    owner: "{{ matrix_user_username }}"
    group: "{{ matrix_user_groupname }}"
  with_items:
    - { directory: "{{ matrix_nginx_proxy_base_path }}" }
    - { directory: "{{ matrix_nginx_proxy_data_path }}" }
    - { directory: "{{ matrix_nginx_proxy_confd_path }}" }
    - { directory: "{{ matrix_nginx_proxy_synapse_cache_path }}", when: matrix_nginx_proxy_synapse_cache_enabled and not matrix_nginx_proxy_enabled }
```
however yamllint check complains for "too many spaces inside spaces". 
Any hint ?

Reply:
changed it to :
```
- name: Ensure Matrix nginx-proxy paths exist
  file:
    path: "{{ item.directory }}"
    state: directory
    mode: 0750
    owner: "{{ matrix_user_username }}"
    group: "{{ matrix_user_groupname }}"
  with_items:
    - directory: "{{ matrix_nginx_proxy_base_path }}"
    - directory: "{{ matrix_nginx_proxy_data_path }}"
    - directory: "{{ matrix_nginx_proxy_confd_path }}"
    - directory: "{{ matrix_nginx_proxy_synapse_cache_path }}"
      when: "matrix_nginx_proxy_synapse_cache_enabled and not matrix_nginx_proxy_enabled"
  ```
  and the checks passed